### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_ExtendyourdataStudio_Keyboard:The install button tool tip is not hoverable. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10642]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#NonBenchmark;#DesktopApp;#MAS1.4.13;#SQL Azure Data Studio;#FTP;#Keyboard;#STP;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.13

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate Extend your data Studio on Welcome screen.
4 . Navigate to install button 

**Actual:**
On hovering over the button install the tooltip appears but on moving to the tooltip it disappears.

**Expected:**
When hovered over the tooltip the tooltip should be hoverable and not dismiss.

**User Impact**:
Keyboard user on trying to move to the tooltip won't be able to view it since it disappears.

**Attachment link for Reference:**
[10642A11y_AzureDataStudioStayHealthy_Welcome_ExtendyourdataStudio_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4705161/10642A11y_AzureDataStudioStayHealthy_Welcome_ExtendyourdataStudio_Keyboard.zip)

**Comment by alanrenmsft** [comment]
@v-bbrady looks like we don't have an active PR to fix this issue.

**Comment by alanrenmsft** [comment]
Hi @fsteffi, the tooltip is standard html tooltip, also, we are updating the tooltip to match the aria-label to be 'Install' so there is no inaccessible information for keyboard users.

there is already an umbrella issue tracking the keyboard accessible tooltip issue: https://github.com/microsoft/vscode/issues/52016

**Comment by Roopesh-T** [comment]
Hi @alanrenmsft the issue, we are not able to interact (Copy the tooltip text, moving the move hover into the tooltip, etc) with the tooltip which appears on hovering with mouse on Install button. 
the issue you mentioned in above comment is about tooltip appearing with keyboard focus. which are 2 different issues.

and the above issue is still repro in the latest Azure Data Studio Insiders application.
![image](https://user-images.githubusercontent.com/86601127/220069406-699994ab-b98f-4b38-92a3-3202ed6df889.png)

![10642_Repro](https://user-images.githubusercontent.com/86601127/220069483-894690f3-150e-44a9-98e5-39b823a47b9d.gif)
Please reopen the bug. the bug should be closed by trusted testers after verificaton.

**GitHub Tags**: #Regressed:02-20-23;

**Comment by kisantia** [comment]
@Roopesh-T can you please test this on the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch)? The tooltip on the "Install" button has been removed.

**Comment by kisantia** [comment]
@Roopesh-T please retest this

**Comment by Roopesh-T** [comment]
closing the bug as the tooltip on the install but got removed.
![image](https://github.com/microsoft/azuredatastudio/assets/86601127/06aad5af-49c5-4731-b2de-f7589366e8f1)
Version: 1.46.0-insider (user setup)
Commit: f8cd52f70a55e3dd510188b65144dbae3cb07cd4
Date: 2023-09-01T05:36:22.643Z
VS Code: 1.79.2
Electron: 22.3.14
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

**Github** : #closed;
